Tools/glib/glib_test_runner.py

```diff
@@ -238,6 +238,12 @@ def _subtest_end(self, result, did_timeout=False):
         self._subtest = None
         self._subtest_messages = []
 
+    def _is_failure_return_code(self, code):
+        # 0 on success, 1 on failure (assuming it returns at all),
+        # 0 or 77 if all tests were skipped or marked as incomplete
+        # See https://gitlab.gnome.org/GNOME/glib/-/blob/main/glib/gtestutils.c#L2361
+        return code != 0 and code != 77
```

Comment:
I find the following a tad more Pythonic, but the current check is fine as well:

```suggestion
        return code not in (0, 77)
```